lib/ansible/modules/files/copy.py

```diff
@@ -45,14 +45,15 @@
   content:
     version_added: "1.1"
     description:
-      - When used instead of 'src', sets the contents of a file directly to the specified value.
+      - When used instead of I(src), sets the contents of a file directly to the specified value.
         This is for simple values, for anything complex or with formatting please switch to the template module.
     required: false
     default: null
   dest:
     description:
-      - Remote absolute path where the file should be copied to. If src is a directory,
-        this must be a directory too.
+      - Remote absolute path where the file should be copied to. If I(src) is a directory, this must be a directory too and
+        I(dest) directory is created when it doesn't exist.
+        If I(src) and I(dest) are files, then parent directory of I(dest) isn't created: the task fails if it doesn't already exist.
     required: true
```

Comment:
Great, thanks for your work.  Good documentation is really valuable.

I think having that sentence at the end works really well.  So while I was thinking about it, I had a final nitpick.  (Sorry!)
"then parent directory" is not quite correct grammar.  I suggest "If I(src) and I(dest) are files, the parent directory of I(dest) isn't created:".